### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dshean**
Did a quick pass through README.  Made some suggestions to streamline some of the text, with minor reorganization.  Will follow up with other thoughts on Slack.  Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,39 +3,34 @@
...
 
-To maximize the utility of this archive, most notebooks, including all those that require large datasets, have been rendered to include outputs. These notebooks can be identified by the "\_rendered" at the end of the filename.
+**Please note that the tutorials presented here used version 0.2.0 of `icepyx`. These tutorials are set up to use that version in the provided Binder link, but they will require modification for more recent versions (>= v0.3.0) of `icepyx`. Up-to-date data access tutorials/examples are available from the [`icepyx`](https://icepyx.readthedocs.io/en/latest/getting_started/example_link.html) repository and associated documentation.**
```
Comment (JessicaS11) [inline README.md]:
```suggestion
**Please note that the tutorials presented here used version 0.2.0 of `icepyx`. These tutorials are set up to use that version in the provided Binder link, but they will require modification for more recent versions (>= v0.3.0) of `icepyx`. Up-to-date data access tutorials/examples are available from the `icepyx`  [repository](https://github.com/icesat2py/icepyx) and [associated documentation](https://icepyx.readthedocs.io/en/latest/getting_started/example_link.html).**
```

File: README.md [hunk 2]
```diff
@@ -3,39 +3,34 @@
...
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ICESAT-2HackWeek/2020_ICESat-2_Hackweek_Tutorials/binder?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252FICESAT-2HackWeek%252F2020_ICESat-2_Hackweek_Tutorials%26urlpath%3Dlab%252Ftree%252F2020_ICESat-2_Hackweek_Tutorials%252F%26branch%3Dbinder)
 
-
 ## Tutorials
 \* Tutorial filenames within this repository denoted below with an asterisk(\*) cannot be run in Binder due to large dataset requirements.
 
 ### 01. Introductory Session (slides)
 *Anthony Arendt and Charley Haley*
-
```
Comment (JessicaS11) [inline README.md:33]: If I recall correctly, I initially didn't have these blank lines here and it interfered with the rendering of the bullets...

**Review by JessicaS11 — CHANGES_REQUESTED** [review]

**Comment by JessicaS11** [comment]
Thanks - these edits look great. I made one minor modification to the icepyx links. Would it be too much to ask you to submit them as an "edits" branch or something? I think you should have push access to the repo, but I can also create one to do a PR against. I've been trying to learn from my mistakes and keep the git history as clean as possible (so not having commits for every edit).

File: README.md (hunk 2 above)
Comment (dshean) [reply README.md:33]: Ah, OK, yeah, I think there are potentially issues around single-item lists.  I think adding two spaces to end of the preceding line might work.  Maybe someday I'll get past the trial and error phase of markdown nuances like this :)

**Comment by dshean** [comment]
Thanks @JessicaS11.  Can you just merge directly to master without creating a new branch?  Hopefully not too much more work required on this.  Can also do a "Squash and Merge" if you're concerned about too many commits.

**Pull request merged**